test/Kontent.Ai.ModelGenerator.Tests/Options/ArgHelpersTests.cs

```diff
@@ -33,7 +33,7 @@ public class ArgHelpersTests
             { "-i", $"{nameof(ManagementOptions)}:{nameof(ManagementOptions.EnvironmentId)}" },
             { "--environmentid", $"{nameof(ManagementOptions)}:{nameof(ManagementOptions.EnvironmentId)}" },
             { "-p", $"{nameof(ManagementOptions)}:{nameof(ManagementOptions.EnvironmentId)}" }, // Backwards compatibility
-            { "--projectid", $"{nameof(ManagementOptions)}:{nameof(ManagementOptions.EnvironmentId)}" } // Backwards compatibility
+            { "--environmentid", $"{nameof(ManagementOptions)}:{nameof(ManagementOptions.EnvironmentId)}" } // Backwards compatibility
```

Comment:
See 2 lines above, i already added the environmentId, and kept the projectId for backwards compatibility